### DISCUSSION TITLE
after registering with oidc first, enable password login

### DISCUSSION THREAD

**Discussion by sshanzel** [ory/kratos#2683] [Q&A]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [X] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe your problem

I have run through quite a lot of pages in the documentation but I can't seem to see a place where it explains how a user who registered through OIDC first has decided to enable password login.

### Describe your ideal solution

Technically I haven't really thought of it. But basically allow password login after registering with OIDC first.

### Workarounds or alternatives

N/A

### Version

0.10.1

**Comment by aeneasr** [comment]
Sign up with OIDC -> Go to your settings -> Set a password

That's it!

**Reply by sshanzel** [reply]
This is actually what I have tried. The update was successful but I can't seem to use it to log in. Do I need to have an additional configuration?

The authentication methods on `/whomai` are only still `oidc`.

![image](https://user-images.githubusercontent.com/13744167/186344183-c028ace5-af7d-45cc-9d2d-7734b38d4188.png)
